### PR TITLE
Add page number to URL in blocks table

### PR DESCRIPTION
**Pull request by natsoni**
https://github.com/mempool/mempool/assets/46578910/d9837be7-8281-46b4-bd73-0792c92b1873

TODO: 

- [x] Pagination on block page
- [x] Use keyboard arrows to navigate in blocks table
- [x] Pagination on liquid pegs table

### CODE REVIEW

**Comment by softsimon** [comment]
Good. Now do Block transactions table ;)

File: frontend/src/app/components/blocks-list/blocks-list.component.ts [hunk 1]
```diff
@@ -52,6 +56,20 @@ export class BlocksList implements OnInit {
 
     if (!this.widget) {
       this.websocketService.want(['blocks']);
+      this.blocksCountInitialized$
+      .pipe(
+        filter(blocksCountInitialized => blocksCountInitialized),
+        take(1)
+      ).subscribe(() => {
```
Comment (softsimon) [inline frontend/src/app/components/blocks-list/blocks-list.component.ts:63]: It's wrong to nest a subscription inside a subscription.
Just use `switchMap() => this.route.queryParams`

File: frontend/src/app/components/blocks-list/blocks-list.component.ts [hunk 2]
```diff
@@ -52,6 +56,20 @@ export class BlocksList implements OnInit {
...
+        this.route.queryParams
+        .pipe(
+          take(1),
+          tap(params => {
+            this.page = +params['page'] || 1;
+            this.pageChange(this.page);
+          })
+        ).subscribe();
```
Comment (softsimon) [inline frontend/src/app/components/blocks-list/blocks-list.component.ts:71]: It's safest to also unsubscribe everything in an ngOnDestroy

**Comment by natsoni** [comment]
Added support for fast table navigation using keyboard arrows: 

https://github.com/mempool/mempool/assets/46578910/8c90a032-6d83-4a02-aa54-8a450c4a33e3

**Review by softsimon — APPROVED** [review]
tested ACK @ [adda51186]

**Pull request merged**